lib/bolt/target.rb

```diff
@@ -95,10 +109,18 @@ def name
       uri
     end
 
+    def remote?
+      @uri_obj.scheme == 'remote' || @protocol == 'remote'
+    end
+
     def port
       @uri_obj.port || @port
     end
 
+    def transport
```

Comment:
I think this could use a discussion of the difference between transport and protocol. Not immediately clear to me what they mean semantically.

Reply:
where should that discussion go?

protocol is part of a url/transport is a bolt thing that is usually the same